home/.vim/config/bundles

```diff
@@ -25,6 +25,8 @@ Bundle 'editorconfig/editorconfig-vim'
 Bundle 'bogado/file-line'
 Bundle 'jceb/vim-orgmode'
 Bundle 'tpope/vim-speeddating'
+Bundle 'tpope/vim-obsession'
+Bundle 'dhruvasagar/vim-prosession'
```

Comment:
`vim-prosession` automates it so you don't have to run any commands, it just does.